frontend/public/components/utils/resource-link.tsx

```diff
@@ -68,49 +72,118 @@ export const resourcePath = (kind: K8sResourceKindReference, name?: string, name
 export const resourceObjPath = (obj: K8sResourceKind, kind: K8sResourceKindReference) =>
   resourcePath(kind, _.get(obj, 'metadata.name'), _.get(obj, 'metadata.namespace'));
 
-export const ResourceLink = connectToModel(
-  ({
-    className,
-    displayName,
-    inline = false,
-    kind,
-    linkTo = true,
-    name,
-    namespace,
-    hideIcon,
-    title,
-    children,
-  }) => {
-    if (!kind) {
-      return null;
+const resourceLinkStateToProps = (state: RootState): resourceLinkStateProps => {
+  return {
+    activeNamespace: state.UI.get('activeNamespace'),
+    receivedCSVs: state.k8s.getIn(['CSVS', 'csvs']),
+  };
+};
+
+export const compareKinds = (csv, owner) => {
+  return (
+    csv.spec.customresourcedefinitions.owned.some((api) => api.kind === owner.kind) &&
+    csv.spec.customresourcedefinitions.owned.some((api) => owner.apiVersion.includes(api.version))
```

Comment:
You don't care about API version here, just name and API group.

This shouldn't be two separate calls to `some`. You want to check that the same item has the right values.

Reply:
I wound up doing this after talking to Tony. The names and api groups don't match easily, unless there's something I'm missing. If you have suggestions, please let me know.

CSV example:
```
{
    ...
    spec:
        apiservicedefinitions: {}
        customresourcedefinitions: {
            owned: 
                [
                    { description: "Represents an installation of a particular version of Knative Serving", displayName: "Knative Serving", kind: "KnativeServing", name: "knativeservings.operator.knative.dev", statusDescriptors: [{…}, {…}], version: "v1alpha1"},
                    {...}
               ]
      }
}
```

Owner example:
```
{
    apiVersion: "operator.knative.dev/v1alpha1"
    blockOwnerDeletion: true
    controller: true
    kind: "KnativeServing"
    name: "knative-serving"
    uid: "fa704434-1ac4-40eb-85b6-ed143a516023"
}
```